src/SFA.DAS.LevyTransferMatching.Web/Helpers/CriteriaListTagHelper.cs

```diff
@@ -17,41 +17,27 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
         {
             var content = new StringBuilder();
             var matchesList = new StringBuilder();
-            int matches = 0;
-            if (Application.IsLocationMatch)
+
+            if (Application.MatchLocation)
```

Comment:
See previous comments about changing this

Reply:
suggestions implemented with as little code changes as needed by this ticket